### DISCUSSION TITLE
Noob help loading a checkpoint

### DISCUSSION THREAD

**Discussion by s-tweed** [coqui-ai/TTS#3709] [General Q&A] [answered]
I spent a few hours training the train_glowtts.py file and got so-so results, so I ended the training and tried out another model. Now if I want to go back and pick up where training left off what is the best way to do that? I'm familiar with python and I know you can load a checkpoint in pytorch, but I know nothing about tensorboard (which is pretty cool btw!) and I'm not sure of the "correct" way to do this without starting my training from scratch.

In a similar vein I'm thinking of starting from some pre-trained checkpoints to reduce training time (I am attempting to clone a specific voice). I assume this would be the same procedure? I just need to find a .pth that I like and load that as the checkpoint?

Thanks! Great repo btw, really clearly put together and I love all of the test code.

- Spencer

**Comment by eginhard** [comment, marked as answer]
See under "Continue a previous run/Fine-tune a model": https://docs.coqui.ai/en/latest/tutorial_for_nervous_beginners.html#pure-python-way

**Reply by s-tweed** [reply]
Thank you, I missed that in the docs!
